### DISCUSSION TITLE
How to disable shift down on ActionButton?

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#2275] [Q&A] [answered]
Hi.
I used ActionButton with className (css module) which have styles like
```css
position: absolute;
  top: 50%;
  transform: translate(0, -50%);
  z-index: 100;
```

I want to disable for that button behavior with shift on active, because it conflicts with this styles.
It is possible somehow?
Maybe can be a good way to introduce a prop?

**Comment by rtivital** [comment, marked as answer]
You can override active styles per component with `sx={{ '&:active: {} }}` and in theme – https://mantine.dev/theming/theme-object/#activestyles
